docs-src/cloud-context/saml-intro.md

```diff
@@ -0,0 +1,24 @@
+---
+id: saml-intro
+title: How to manage SAML authentication with Temporal Cloud
+sidebar_label: Manage SAML
+description: To authenticate users of your Temporal Cloud account, you can connect a SAML IdP.
+tags:
+  - guide-context
+---
+
```

Comment:
Do we need to notify customers to open a ticket to enable SAML?

Reply:
If that's the mechanism, I can definitely add it, with a link to https://docs.temporal.io/cloud/how-to-create-a-ticket-for-temporal-support.